### ISSUE TITLE
How to get the fully connected layer feature before the bbox predictions?

### ISSUE THREAD

**Issue by XinliYu** [facebookresearch/detectron2#1676]
My input is an image or a batches of images.

I would like the fully connected layer output before the FastRCNN OutputLayers and use it as pre-trained features. 
I am trying to modify the partially executing model way in the tutorial to get it.
```
model = build_model(cfg)
features = model.backbone(images.tensor)
proposals, _ = model.proposal_generator(images, features)
<I guess insert it here, trying to use the forward function here in box_head>
instances = model.roi_heads._forward_box(features, proposals)
mask_features = [features[f] for f in model.roi_heads.in_features]
mask_features = model.roi_heads.mask_pooler(mask_features, [x.pred_boxes for x in instances])

```
I did some research and I found I need to use FastRCNNConvFCHead part, however, the forward function there seems not callable in evaluation mode. 
```
 def forward(self, x):
        for layer in self.conv_norm_relus:
            x = layer(x)
        if len(self.fcs):
            if x.dim() > 2:
                x = torch.flatten(x, start_dim=1)
            for layer in self.fcs:
                x = F.relu(layer(x))
        return x
```
Do you have any suggestions how to modify it to get the fully connected layer? Thanks a lot for help!

**Comment by XinliYu** [comment]
I got how to do it. Use the box_pooler and box_head will do it. Thanks!
